testing/scripts/metrics.yaml

```diff
@@ -0,0 +1,138 @@
+kind: ClusterRole
```

Comment:
nit: should we move `metrics.yaml` to the `./testing/resources/` folder?

Reply:
Not sure as we still have istio-1.4.2.yaml in this folder. These two yamls are part of the cluster setup.